CONTRIBUTING.md

```diff
@@ -17,28 +17,45 @@ The following is a set of guidelines for contributing to the UofT Coders communi
     - [Other Ways to Get Involved](#other-ways-to-get-involved)
 
 -----
-## What You Should Know
+# What You Should Know
 
 ### Code of Conduct
 
-We adhere to a [Code of Conduct](https://github.com/UofTCoders/studyGroup/blob/gh-pages/codeOfConduct.md) and by participating, you agree to also uphold this code. Please report any unacceptable behaviour to uoftcoders@gmail.com. If you feel that the exec has been violating the Code of Conduct, please email groups.officer@utoronto.ca to report. 
+We adhere to a [Code of Conduct](https://github.com/UofTCoders/studyGroup/blob/gh-pages/codeOfConduct.md) and by participating, you agree to also uphold this code. Please report any unacceptable behaviour to uoftcoders@gmail.com. If you feel that the exec has been violating the Code of Conduct, please email groups.officer@utoronto.ca to report.
 
 ### About Uoft Coders
 
-The UofT Coders group was formed to share and learn about coding techniques and best practices for computing and analysis in research. We hold weekly sessions in the format of code-alongs, coworking sessions or journal clubs/discussions. To see our previous and upcoming events, you can visit our [Events repo](https://github.com/UofTCoders/Events/issues). To learn more about our group, you can read our [constitution](https://github.com/UofTCoders/council/blob/master/Constitution.md). 
+The UofT Coders group was formed to share and learn about coding techniques and best practices for computing and analysis in research. We hold weekly sessions in the format of code-alongs, coworking sessions or journal clubs/discussions. To see our previous and upcoming events, you can visit our [Events repo](https://github.com/UofTCoders/Events/issues). To learn more about our group, you can read our [constitution](https://github.com/UofTCoders/council/blob/master/Constitution.md).
 
 -----
```

Comment:
I think the headings could be restructured like so for clarity 

- About UofT coders
    - Code of Conduct
- How you can contribute
    - Teaching/Giving/Making/Leading a lesson
        - Creating the content
        - Teaching in class
        - Submitting the lecture as a Pull Request
            - Brief step by step
            - Detailed step by step
    - Updating the website
    - Helping out at our events
    - Taking on a leadership role

I am tempted to put the detailed step by step instruction for the PR in a separate document and link to it, just to make this main document short and straightforward. Thoughts @linamnt @QuLogic ?

Reply:
@joelostblom that makes sense, though I'm not sure where you would put it. Maybe like "./_contributing/pull-requests.md" or something? (the `_` is to prevent Jekyll building of it... unless we want this contributing docs on the website too..? I think that would be cool.)

Reply:
Yea we could put it on the website, I don't see why not?